### DISCUSSION TITLE
Run In Drive Removed

### DISCUSSION THREAD

**Discussion by tikilab** [acheong08/Diffusion-ColabUI#80] [Q&A] [answered]
I've been using Diffusion WebUI running on my Google Drive (even though it recommends not to) because I've been using the same model and wildcards, etc and they are saved, and I haven't had any issues. But I guess it's been updated to only save the images on my Google drive now. (I have been regularly reinstalling it to get the latest updates.) Is there any way running on the Google drive could be brought back? I have an old Mac and this is the easiest (i.e. cheapest) way for me to run SD.

**Comment by acheong08** [comment, marked as answer]
You can use [colab.research.google.com/github/acheong08/Diffusion-ColabUI/blob/old/Diffusion_WebUI.ipynb](https://colab.research.google.com/github/acheong08/Diffusion-ColabUI/blob/old/Diffusion_WebUI.ipynb) (which still supports gdrive)

**Reply by acheong08** [reply]
There were error reports about issues with Google Drive and it was therefore taken down as a feature from the main branch. I will keep it maintained in a separate branch

**Reply by tikilab** [reply]
Great, thank you!
